cwltool/schemas/v1.0/Process.yml

```diff
@@ -562,7 +562,9 @@ $graph:
 
     - name: default
       type: Any?
-      jsonldPredicate: "cwl:default"
+      jsonldPredicate:
```

Comment:
Hi @mr-c  Do you mean I should send a PR on https://github.com/common-workflow-language/common-workflow-language/ . Also should I make changes in v1.1.0-dev1 or v1.0
Thanks